cursos/helm/00_solucions/99_other_wp/actividade-1-charts-helm/wp-release/templates/wp-deploy.yaml

```diff
@@ -0,0 +1,39 @@
+# wp-deploy.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deploy-wp
+  labels:
+    app: "deploy-wp"
+    tipo: "practica"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: deploy-wp
+  template: 
+    metadata:
+      labels:
+        app: deploy-wp
+    spec:
+      containers:
+      - name: deploy-wp
+        image: wordpress:6.1
+        env:
+          - name: "MYSQL_PASSWORD"
+            valueFrom:
+              secretKeyRef:
+                name: wp-secrets
+                key: root-password
+        envFrom:
+          - configMapRef: 
+              name: wp-config
+        ports:
+        - containerPort: 8080
+        volumeMounts:
+        - name: wordpress-persistent-storage
+          mountPath: /var/www/html
```

Comment:
I think the mount path should be the folder with the files that will be generated during the normal operation of the site, in specific the `wp-content/uploads` path right?, not all the html folder, that will come with the docker image

https://www.theistudio.com/where-are-wordpress-files-stored/#h-wordpress-file-locations

Reply:
It would be more correct from wp-content to add the layout, theme and so on. But yes, from the html directory would be too much.